### PR TITLE
Fix name of color_sequence.dart and add a test

### PR DESCRIPTION
**Pull request by eseidelGoogle**
@hixie @vlidholt

### CODE REVIEW

File: packages/flutter_sprites/tests/color_sequence_test.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import 'dart:ui';
+import 'package:test/test.dart';
+import 'package:flutter_sprites/flutter_sprites.dart';
+
+void main() {
+  test("Simple test of ColorSequence", () {
+    List<Color> colors = [const Color(0xFFFFFFFF), const Color(0x000000FF)];
```
Comment (Hixie) [inline packages/flutter_sprites/tests/color_sequence_test.dart]: `<Color>[...]`

File: packages/flutter_sprites/tests/color_sequence_test.dart [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    List<double> stops = [0.0, 1.0];
```
Comment (Hixie) [inline packages/flutter_sprites/tests/color_sequence_test.dart]: `<double>[...]`

**Comment by Hixie** [comment]
Please wait until https://github.com/flutter/flutter/pull/912 is fixed before merging this (and rerun travis before doing so, to make sure there's no new warnings).

File: travis/test.sh [hunk 3]
```diff
@@ -10,7 +10,7 @@ flutter analyze --flutter-repo --no-current-directory --no-current-package --con
 flutter test --flutter-repo
 
 (cd packages/cassowary; pub run test -j1)
-# (cd packages/flutter_sprites; ) # No tests to run.
+(cd packages/flutter_sprites; flutter test) # No tests to run.
```
Comment (abarth) [inline travis/test.sh]: This comment seems out of date now.  :)

**Comment by eseidelGoogle** [comment]
Filed https://github.com/dart-lang/sdk/issues/25254 about the <Color> <int> thing.

**Comment by eseidelGoogle** [comment]
Looks like it can't find flutter_sprites in the packages directory?  I'm not sure how the packages directory is controlled on Travis. @Hixie

**Comment by Hixie** [comment]
https://github.com/flutter/flutter/pull/916 should fix the flutter_sprites-not-found issue.

**Comment by Hixie** [comment]
I don't understand why travis is unhappy.

File: travis/test.sh [hunk 4]
```diff
@@ -10,7 +10,7 @@ flutter analyze --flutter-repo --no-current-directory --no-current-package --con
 flutter test --flutter-repo
 
 (cd packages/cassowary; pub run test -j1)
-# (cd packages/flutter_sprites; ) # No tests to run.
+(cd packages/flutter_sprites; flutter test)
```
Comment (Hixie) [inline travis/test.sh]: this won't be necessary, `flutter test -flutter-repo` will cover this.

Comment (Hixie) [inline travis/test.sh]: actually now none of these will be necessary for that reason.

File: packages/unit/pubspec.yaml [hunk 5]
```diff
@@ -5,6 +5,8 @@ dependencies:
   flutter:
     path: ../flutter
   flutter_test:
+    path: ../flutter_sprites # Hack until @hixie fixes flutter analyze
+  flutter_test:
```
Comment (Hixie) [inline packages/unit/pubspec.yaml]: you should be able to remove this now.

File: travis/test.sh (hunk 4 above)
Comment (Hixie) [inline travis/test.sh]: I take this back. I tried to add this to flutter test but flutter test really wants to run in just one package at a time. We could make it more generic but there's really no reason to.

**Comment by eseidelGoogle** [comment]
I'm not sure why travis is unhappy either.  I'm unlikely to finish this CL before January.  Please feel free to take it over or work around it.  @devoncarew

**Comment by Hixie** [comment]
Closing this since Devon took it over in #1043.

**Pull request closed without merging**